Bulldozer.CSV/Maps/ConnectionRequest.cs

```diff
@@ -126,7 +126,22 @@ private int LoadConnectionRequest( CSVInstance csvData )
 
                     if ( request == null && requestor != null && requestStatus != null )
                     {
-                        request = AddConnectionRequest( opportunity, rForeignKey, rCreatedDate, rModifiedDate, requestStatus.Id, ( ConnectionState ) rState, rComments, rFollowUp, requestor.PersonAliasId, requestConnector?.PersonAliasId );
+                        request = new ConnectionRequest
```

Comment:
remove original method now that it is not being used.

Reply:
Had to update F1 notes map to same structure to be able to get rid of the method